app/(white_navbar)/explore/RetroPGF5/[name]/_component/_Project/ImpactStatementSection.tsx

```
@@ -45,25 +45,23 @@ export default function ImpactStatementSection({
+        data.impactStatement.statement?.map((item: Statement, i: number) => (
+          <div key={i} className="flex flex-col gap-y-4">
+            <div className="text-lg break-words font-semibold text-gray-800">
+              {item.question}
             </div>
-          )
-        )}
+            <div>
+              {item.answer && (
+                <a
+                  className="text-base break-words font-normal text-gray-600 space-y-5"
+                  dangerouslySetInnerHTML={{
```

Biome

[error] Avoid passing content using the dangerouslySetInnerHTML prop.

Setting content using code can expose users to cross-site scripting (XSS) attacks

(lint/security/noDangerouslySetInnerHtml)